packages/iov-faucets/src/bovfaucet.spec.ts

```diff
@@ -0,0 +1,32 @@
+import { Address, TokenTicker } from "@iov/bcp-types";
+import { Random } from "@iov/crypto";
+
+import { BovFaucet } from "./bovfaucet";
+
+describe("BovFaucet", () => {
```

Comment:
Seems a bit funny for unit tests to run against a live network, but there would be too much overhead to actually set up and run a local faucet, right?

We could start up https://hub.docker.com/r/iov1/faucet/tags/ but I will leave this as an open issue so we don't depend on friendnet so much.